ntoskrnl/config/cmsysini.c

```diff
@@ -1329,8 +1336,19 @@ CmpLoadHiveThread(IN PVOID StartContext)
                 //ASSERT(FALSE);
             //}
 
-            /* Another thing we don't support is NTLDR-recovery */
-            if (CmHive->Hive.BaseBlock->BootRecover) ASSERT(FALSE);
+            /*
+             * FreeLdr has recovered the hive. We have
+             * to flush every dirty data that's been
+             * written to the hive by FreeLdr to
+             * the backing storage.
+             */
```

Comment:
you are wasting many lines with your commenting style of "beginning and ending each comment on its own line".

Reply:
Have you bothered to check other parts of the kernel where multi-line comments (e.g. https://github.com/reactos/reactos/blob/master/ntoskrnl/ob/obname.c#L234 and etc) before calling it "your commenting style" before reviewing?

Reply:
Same thing: Other places do not comment as verbosely as you do. I do not really care about the style of your commenting, but I am ciriticizing the sheer amount of comments in relation to the code in your PR. So I added some specific suggestions about how you could improve. You can chose other ways to reduce their amount, I don't really care how you decide to improve, I do just point to the fact that you comment too excessively.
You should ask yourself the question for each and every comment that you put into your code:
Does it really help others to better understand the code? Does it really add some value? Or is it it just saying the same thing that can be easily seen by anybody just looking at the code directly.

Writing too many comments is a bad practice in the same way as writing too much code is. For the same reason: both is error-prone and each line you write today will get viewed/reviewed/changed and later updated a thousand times by others. So what you add, should really offer some value.
It is definitely no good idea to comment each and every line of code you write!